### DISCUSSION TITLE
IPv6 support for Micropython on RP2040 (Pico W)

### DISCUSSION THREAD

**Discussion by nikchosh** [micropython/micropython#11180] [RP2040 / Pico] [answered]
Hi! I see there is LWIP_IPV6=0 (disabled) in the https://github.com/micropython/micropython/blob/master/ports/rp2/lwip_inc/lwipopts.h
Is there a possibility to enable it generally, or is it disabled due to performance/space constraints?
As the underlying driver supports it, should I attempt (quite noob) to build MP for RPI Pico W with IPV6 setting on, and see what happens?

**Comment by mattytrentini** [comment, marked as answer]
Keep an eye on #9108 which intends to add IPv6 for the Pico W. If you can help test, that'd be great!
